int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -460,47 +479,49 @@ function edqSetProWebConfiguration() {
 		}
 	];
 	window.EdqConfig.PRO_WEB_LAYOUT= edqProWebAddressLayout;
-	if (edqCountryLineSelector != null)
+	if (edqCountryLineSelector != null) {
 		window.EdqConfig.PRO_WEB_COUNTRY= countryAlpha3(edqCountryLineSelector.value);
-	else
+	}
+	else {
 		window.EdqConfig.PRO_WEB_COUNTRY= countryAlpha3(vDefaultCountry);
+	}
 	window.EdqConfig.PRO_WEB_CALLBACK= "edqValidateAddressCallBack";
 	window.EdqConfig.PRO_WEB_MAPPING= [
 		{
 			field: edqAddressLine1Selector,
-			elements: ["Formatted Address 2"],
+			elements: ["address.addressLine1"],
```

Comment:
This is not correct. You're using Global Intuitive element names. Did you mean to do this? What layout does thsi work with?

Reply:
The layout name is "EDQ NA Integration 5 Line Default" and the layout element names were intended to be the same as Global Intuitive element names

Reply:
Ah, interesting. OK sounds good then.